### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Anoerak** [babybuddy/babybuddy#565] [Q&A]
Hi,
 
I try to edit the "fr" translation file to correct some boo-boo and add a translation for the "recent sleep" title card. 
I don't know what I'm doing wrong, my modifications don't appear in my development environment 🤔. 
Would love some help to understand what I'm doing wrong here. 
Thanks.

**Comment by cdubz** [comment]
After you make a change you need to run `gulp compilemessages -l fr` to regenerate the PO files with your changes.

There is some additional documentation about translations here as well: https://docs.baby-buddy.net/contributing/translation/

**Comment by Anoerak** [comment]
Thanks 🙏🏼
